guide/voice/optimisation-and-troubleshooting.md

```diff
@@ -62,7 +62,7 @@ And that's it! discord.js will not create an FFmpeg transcoder for your file and
 
 #### From YouTube videos
 
-Many voice bots allow an ability to play audio from YouTube videos in voice channels. YouTube itself provides WebM/Ogg streams for newer videos, and so we can also demux these files for Opus audio instead of running them through an FFmpeg transcoder first.
+Many voice bots allow an ability to play audio from YouTube videos in voice channels. YouTube itself provides WebM/Ogg streams for newer videos, and so you can also demux these files for Opus audio instead of running them through an FFmpeg transcoder first.
```

Comment:
```suggestion
Many voice bots have the ability to play audio from YouTube videos in voice channels. YouTube itself provides WebM/Ogg streams for newer videos, which you can demux for Opus audio instead of running them through an FFmpeg transcoder first.
```